functions/Get-DbaClusterSqlInstance.ps1

```diff
@@ -0,0 +1,84 @@
+function Get-DbaClusterSqlInstance
+{
+<#
+.SYNOPSIS
+Returns the registered SQL Server Instances on a windows Server cluster
+	
+.DESCRIPTION
+By default, this command returns for each Cluster passed in:
+Pulls Cluster details via WMI/CIM so doesn't require Failover Cluster PowerShell module to be installed.
+Multiple clusters can be piped in.
+Returns ALL instance registered on cluster. Does not check that they are viable, correctly installed or currenly running. 
+	
+.PARAMETER Cluster
```

Comment:
Yes, that's what I meant. Not multiple, but sometimes computer1, and sometimes computer2. So still, should we describe that as "the sql server", while the parameter's name is "cluster"